INIT.md

```diff
@@ -1,25 +1,77 @@
-## webpack init
+# webpack-cli init
 
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
 
-We ask several questions in the default generator to get you started.
+## Initial Setup
+A. **Local setup**
 
----
+Follow given steps to locally setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+	$ npm init
+	```
+
+2. Install `webpack` and `webpack-cli` as devDependencies
+
+	```shell
+    $ npm install --save-dev webpack webpack-cli
```

Comment:
I'd be consistent and would write `--save-dev` or `-D`, not both. They are basically the same. `-D` is just an alias

Reply:
Updated it @ematipico